src/app/dynamic-forms/cinchy-dynamic-forms.component.ts

```diff
@@ -574,37 +672,104 @@ export class CinchyDynamicFormsComponent implements OnInit, OnChanges, OnDestroy
             // So it was not adding any new row in the child table when user added a row just after save from + icon, during new insert
             // this.openChildDialog(childData);
           }
-        }, 500)
+        }, 500);
       } catch (e) {
         this.spinner.hide();
         if (e && e.cinchyException && e && e.cinchyException.message) {
           console.error(e.cinchyException.message);
-          this._toastr.error('Access denied on table or Error while getting table data.', 'Error')
+          this._toastr.error('Access denied on table or Error while getting table data.', 'Error');
         } else {
-          const message = currentRowItem ? e : "Selected row is either Deleted or doesn't exist in Cinchy";
-          this._toastr.error(message, 'Error')
+          const message = currentRowItem ? e : 'Selected row is either Deleted or doesn\'t exist in Cinchy';
+          this._toastr.error(message, 'Error');
         }
       }
     }
+
     this.formsData = result;
-    //   console.log(result);
-    // this.form = result;
     return result;
   }
 
+  public cancel(pendingcall: any) {
+    if(this.pendingcall != null) {
+      // this.pendingcall.unsubscribe();
+      // cancel the request
+      this.cinchyQueryService.stopRequest.next();
+      console.log('unsubscribed');    
+    }
+  }
+
+    async getJsonDataSection(expand: boolean, formID?, sectionId?) {
+    let JsonDataNew;
```

Comment:
can we make the first letter of all these variables within the scope of the function lowercase?

Reply:
Resolved